test/addons-napi/test_exception/test.js

```diff
@@ -12,7 +12,8 @@ let caughtError;
 // Test that the native side successfully captures the exception
 let returnedError = test_exception.returnException(throwTheError);
 assert.strictEqual(theError, returnedError,
-                   'Returned error is strictly equal to the thrown error');
+                   `Returned error ${returnedError.message}` +
+                   ` is strictly equal to the thrown error ${theError.message}`);
```

Comment:
Linter complained:

```
not ok 23 - /usr/home/iojs/build/workspace/node-test-linter/test/addons-napi/test_exception/test.js
  ---
  message: Line 16 exceeds the maximum line length of 80.
  severity: error
  data:
    line: 16
    column: 1
    ruleId: max-len
  ...
```

I think we can just remove the assertion message (third argument) here, that would actually be more readable.